static/redfish/v1/$metadata/index.xml

```diff
@@ -2829,4 +2829,8 @@
         <edmx:Include Namespace="OemSession"/>
         <edmx:Include Namespace="OemSession.v1_0_0"/>
     </edmx:Reference>
+        <edmx:Reference Uri="/redfish/v1/schema/OemLogEntry_v1.xml">
```

Comment:
This is the CI fail 

diff --git a/static/redfish/v1/$metadata/index.xml b/static/redfish/v1/$metadata/index.xml
index 43ffd0fe..f6f2d742 100644
--- a/static/redfish/v1/$metadata/index.xml
+++ b/static/redfish/v1/$metadata/index.xml
@@ -2829,7 +2829,7 @@
         <edmx:Include Namespace="OemSession"/>
         <edmx:Include Namespace="OemSession.v1_0_0"/>
     </edmx:Reference>
-        <edmx:Reference Uri="/redfish/v1/schema/OemLogEntry_v1.xml">
+    <edmx:Reference Uri="/redfish/v1/schema/OemLogEntry_v1.xml">
         <edmx:Include Namespace="OemLogEntry"/>
         <edmx:Include Namespace="OemLogEntry.v1_0_0"/>
     </edmx:Reference>